src/components/WorkOrder/Authorisation/__snapshots__/VariationAuthorisationSummary.test.js.snap

```diff
@@ -122,7 +122,7 @@ exports[`VariationAuthorisationSummary component should render properly 1`] = `
       <p
         class="lbh-body"
       >
-        Tuesday, 11 May 2021 
+        Tuesday 11 May 2021 
```

Comment:
Is the missing comma intentional?

_(I know it's a snapshot, but it's a result of a render, so wherever the actual code is, it misses a comma)_

Reply:
its due to node version update. doesnt matter enough for me to fix it